en/06_Project_Governance/03_Maintainer_Guidelines.md

```diff
@@ -36,23 +40,113 @@ often working alongside Core Committers.
 
```

Comment:
I'm not sure what this comment is for... is this a note for yourself to review this again? Or are you requesting a change? Or...?

Reply:
Done, but with "leader" instead of "lead"

Reply:
I really don't think this document cares about job titles. I'll update mentions of the "CMS squad team leader" (i.e. leader of the CMS squad) to mention instead the "Silverstripe CMS Product Owner" since that's really the relevant role for those mentions.